sdk/identity/azure-identity/README.md

```diff
@@ -159,7 +118,14 @@ If the environment configuration is not present or incomplete, the `DefaultAzure
 require platform support. See the
 [managed identity documentation](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities) for more details on this.
 
-If a managed identity isn't available, and the application is running on Windows, the `DefaultAzureCredential` will then attempt reading from a local user token cache. `AZURE_USERNAME` environment variable must be specified if there are more than one accounts in the cache. The local token cache is shared between this library, Visual Studio (2019+), and Azure CLI. See [Enable applications for shared token cache credential](#enable-applications-for-shared-token-cache-credential) to see how to populate / clean up this token cache.
+If a managed identity isn't available, the `DefaultAzureCredential` will then attempt reading the signed in user from a set of local developer tools, in the following order
```

Comment:
Should we also mention about EnvironmentCredential not being available too  and then attempting below chain of creds ?

Reply:
It's mentioned in line 117 - just not shown in GitHub diff